src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/CcdCaseRetriever.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+@Service
+public class CcdCaseRetriever {
+    public static final String CASE_TYPE_ID = "Bulk_Scanned";
+
+    private CcdAuthenticatorFactory authenticator;
+    private final CoreCaseDataApi coreCaseDataApi;
```

Comment:
why is this final and not the authenticator factory?

Reply:
sure its a refactoring miss :D 
Fixing.